config/routes.rb

```diff
@@ -42,6 +42,11 @@ def matches?(request)
       get     "/resources/:account"                   => "resources#index"
       get     "/resources"                            => "resources#index"
 
+      # NOTE: the order of these routes matters: we need the expire 
+      #       route to come first.
+      constraints account: /[^\/]*/, kind: /[^\/]+/, identifier: /.+/ do
+        post    "/secrets/:account/:kind/:identifier/expirations" => "secrets#expire"
```

Comment:
When we originally talked about this route, I guess I was expecting that `POST`ing to `/secrets/:account/:kind/:identifier/expirations` was going to take an expiration from the client and apply it to the secret.

Since that's not what's happening, I wonder if this should be something else (though I'm not sure what)?

Reply:
@apotterri that was discussed, this was the feature we ended up going with per @jvanderhoof 's note that this was what was needed and trying to small batch it.  adding general expiration later should be close to trivial.

Reply:
If possible, can we keep this change inline with the current v4 functionality?  It'll make documentation & education a lot easier.